pydicom/dataelem.py

```diff
@@ -775,47 +775,30 @@ def _private_vr_for_tag(ds: Optional["Dataset"], tag: BaseTag) -> str:
     return "UN"
 
 
-def DataElement_from_raw(
-    raw_data_element: RawDataElement,
-    encoding: Optional[Union[str, MutableSequence[str]]] = None,
-    dataset: Optional["Dataset"] = None
-) -> DataElement:
-    """Return a :class:`DataElement` created from `raw_data_element`.
+def pydicom_raw_elem_preprocess(
+    raw: RawDataElement,
```

Comment:
I don't think 'pydicom' needs to be in the names, its all pydicom - but do like something with 'raw' in it because it's part of that process.  Perhaps `raw_convert_VR`,  or `raw_check_VR`, something along those lines